### DISCUSSION TITLE
Website location in  functions

### DISCUSSION THREAD

**Discussion by PerroniClaudio** [vercel/community#2389] [Help] [answered]
### Summary

Good morning. 

I did follow the guide to change the location of the function (project -> settings ->function -> change the region to frankfurt), however if i use a geolocator plugin for my browser it still appears in california. 

I need some assistance because EU laws require the website to be hosted in the EU, and it can't show anything outside the EU (even london is problematic)

I did redeploy as instructed, the result did not change

### Example

https://www.le-news.it/

**Comment by amyegan** [comment, marked as answer]
Hi @PerroniClaudio. You can find more information about hosting location and related factors in the following documentation:
- [Does my Domain IP address on Vercel resolve to a geographic region?](https://vercel.com/guides/domain-ip-address-geographic-location)
- [Security & Compliance Measures](https://vercel.com/docs/security#)
  - [GDPR](https://vercel.com/docs/security#gdpr)
  - [Where does my data live?](https://vercel.com/docs/security#where-does-my-data-live)
